SK.WebApp/src/store/session/sessionSlice.ts

```diff
@@ -0,0 +1,47 @@
+import { createSelector, createSlice } from "@reduxjs/toolkit";
+import { RootState } from '..';
+
+interface SessionState {
+  username: string;
+  token: string;
+  image: string;
+};
+
+const initialState : SessionState = {
+  username: "",
+  token: "",
+  image: "",
+};
+
+export const sessionSlice = createSlice({
```

Comment:
As I said somewhere up there, really nice use of redux toolkit!

Reply:
thanks